articles/article-088-no-free-lunch.md

```diff
@@ -0,0 +1,25 @@
+<!-- BEGIN TITLE -->
+# There is no such thing as a free lunch
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In Computer Science we have  a wide range of very different problems and accordingly many approaches to tackle them.
```

Comment:
In computer science, ...
- do not capitalize things that are not proper nouns 
- comma after prepositional phrases

Reply:
you can just say:
...very difficult problems and many approaches to tackle them. 
- the accordingly is unnecessary